### ISSUE TITLE
Essential+MixedModeAuth = Server Fails To Ping

### ISSUE THREAD

**Issue by Favorlock** [Thulinma/MixedModeAuth#3]
So my friend uses a hosting manager called Multicraft, but it would seem that when essentials and MixedModeAuth are enabled together the manager will restart claiming that the server cannot be pinged or loses connection yada yada. I also tend to see some messages in the multicraft console saying x number of players with some very weird syntax. I don't see these things in a command prompt, but in the manager I do. So apparently it's more of a Multicraft+Essentials+MixedModeAuth issue. If you could look into this I'd appreciate it.

**Comment by Thulinma** [comment]
Interesting.
I'll see if I can reproduce this - is MixedModeAuth (trying to be) running in legacy or modded mode? And what version of Essentials do I need to try this with?

**Comment by Favorlock** [comment]
MixedModeAuth is running in modded mode. I used the version 2.6.4 of essentials.

**Comment by Thulinma** [comment]
Is this still happening with the latest release?
Some related code was changed and this might be fixed now... I didn't test it myself yet though.

**Comment by Favorlock** [comment]
Yes, the problem is still persistent on the latest developer build.

**Comment by Thulinma** [comment]
Bah, it was too good to be true. This might be related to issue #4 then, since that deals with names not working right, and I assume Essentials does something with the usernames, somewhere :-)

Does this also happen without Multicraft, by the way? Does it happen without players online? After players sign on? Online or offline mode players? The more extra info you can provide, the better!

**Comment by Favorlock** [comment]
I know that MixedModeAuth incorporates some sort of syntax that Multicraft can't manage, so the names messed up in Multicraft are likely caused by MixedModeAuth. No, none of this happens when it's just the two plugins without Multicraft, however, there are no conflicts when it is just Multicraft and Essentials, that's how I deduced that it was the combination of all three. It's primarily because it's causing Multicraft to restart over, and over due to the ping or something.

It starts as soon as the server starts, regardless whether players are online. I keep the server in online mode due to MCBans. Other than that I really can't think of anything that would cause any problems.

**Comment by Thulinma** [comment]
Alright, I must be missing something in the player renaming code. I'll compare to the CraftBukkit code and see if I can get this fixed...
